### PR TITLE
fix: create cloud org modal

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes  UNIFY-1084

### User facing changelog

- Because we're using cache-first it's avoiding doing the extra requests to the cloud - so, in this case, we need to have the data up to date. -> users will see new organizations when they create them.
- Since the actual requests are quite slow and could be costly, we slow down the rhythm of repeat requests. Polling is now every 5 seconds.

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by elevatebart** [comment]
@estrada9166 Can you explain what you did and why?

The only changes I see are:
- change timeouts
- use `network-only` for the query

**Comment by estrada9166** [comment]
@elevatebart sure! so, what's going on is that because we're using `cache-first` it's avoiding doing the extra requests to the cloud - so, in this case, we need to have the data up to date.

And related with the timeouts, it was to reduce the amount of requests that can be done while creating the org

**Review by elevatebart — APPROVED** [review]
Thank you for the explanations, I will try to add them to the description

**Review by elevatebart — DISMISSED** [review]

File: packages/app/src/runs/modals/CreateCloudOrgModal.vue [hunk 1]
```diff
@@ -123,7 +124,7 @@ function startPolling () {
   setTimeout(() => {
     pause()
     polling.value = false
-  }, 180000)
+  }, 60000)
```
Comment (tbiethman) [inline packages/app/src/runs/modals/CreateCloudOrgModal.vue:127]: Does the polling continue if the modal is dismissed? Should we be pausing/clearing the interval when this component is unmounted?

Comment (elevatebart) [reply packages/app/src/runs/modals/CreateCloudOrgModal.vue:127]: We should... I thought I did TBH.

Comment (estrada9166) [reply packages/app/src/runs/modals/CreateCloudOrgModal.vue:127]: yeah, if the modal is closed it stops polling

**Review by elevatebart — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

**Review by tgriesser — APPROVED** [review]

**Pull request merged**